src/components/projecteditor/panels/explorer/items/style.less

```diff
@@ -26,7 +26,7 @@ select {
     user-select: none;
     display: block;
     color: #fff;
-    line-height: 28px;
```

Comment:
Explorer feels too much compact on non-retina display. I would consider to have line-height at least 24-25px.

Reply:
Sure. In fact, I thought it did have 24px but let me force it instead through this prop.